source/utilities.tex

```diff
@@ -7110,6 +7112,44 @@
 arguments for the same buffer.  \end{note}
 \end{itemdescr}
 
+\indexlibrary{\idxcode{assume_aligned}}%
+\begin{itemdecl}
+template<size_t N, class T>
+  [[nodiscard]] constexpr T* assume_aligned(T* ptr);
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\mandates
+\tcode{N} is a power of two.
+
+\pnum
+\effects
+\tcode{ptr} points to an object \tcode{X} of
+a type similar\iref{conv.qual} to \tcode{T},
+where \tcode{X} has alignment \tcode{N}.
```

Comment:
That wasn't in the ingress paper, but please add a cross-reference to [basic.align] after "N" (this is the core language section on alignment).

Reply:
done